tools/mo/openvino/tools/mo/convert_impl.py

```diff
@@ -873,12 +873,6 @@ def _convert(cli_parser: argparse.ArgumentParser, framework, args, python_api_us
                 elif 'example_inputs' in args:
                     raise AssertionError("'example_inputs' argument is not recognized, maybe you meant to provide 'example_input'?")
 
-                if 'use_legacy_frontend' in args and args['use_legacy_frontend']:
```

Comment:
We have tests for this functionality in https://github.com/openvinotoolkit/openvino/blob/eb178a753b5f0536b19efe597e805ef6634f16c6/tests/layer_tests/mo_python_api_tests/test_mo_convert_pytorch.py#L104

Please remove **use_legacy_frontend** flag in tests